kubernetes/fluentd-daemonset-filtered.yaml

```diff
@@ -1,115 +0,0 @@
-apiVersion: extensions/v1beta1
```

Comment:
This is part of the tutorial here: https://cloud.google.com/solutions/customizing-stackdriver-logs-fluentd.  Are you proposing a change to that tutorial?

Reply:
Since the only change in this file was the name of the ConfigMap, I'm thinking about adding this change as a manual step in the tutorial so it is more clear to the customer. What do you thing about it?